app/domain/errors.rb

```diff
@@ -420,22 +420,47 @@ module AuthnJwt
         code: "CONJ00178E"
       )
 
-      InvalidUriConfiguration = ::Util::TrackableErrorClass.new(
-        msg: "Signing key uri authenticator configuration is invalid. It should configured as authenticator variables: " \
-              "one of the following: '{0-resource-name}','{1-resource-name}'",
+      FetchJwksKeysFailed = ::Util::TrackableErrorClass.new(
+        msg: "Failed to fetch jwks from '{0-uri}'. Reason: '{1}'",
         code: "CONJ00179E"
       )
 
-      FetchJwksKeysFailed = ::Util::TrackableErrorClass.new(
-        msg: "Failed to fetch jwks from '{0-uri}'. Reason: '{1}'",
+      FetchJwksUriKeysNotFound = ::Util::TrackableErrorClass.new(
+        msg: "Fetched jwks not found in response: '{0-base64-response}'",
         code: "CONJ00180E"
       )
 
-      FetchJwksUriKeysNotFound = ::Util::TrackableErrorClass.new(
-        msg: "Fetched jwks not found in response: {0}",
+
+      UnsupportedClaim = ::Util::TrackableErrorClass.new(
+        msg: "Claim '{0-claim}' is unsupported to fetch the application identity.",
         code: "CONJ00181E"
       )
 
+      MissingClaimValue = ::Util::TrackableErrorClass.new(
```

Comment:
Do you mean that "there the claim value is empty, or the claim is not found"?
Or do you mean that the value of the claim is not found?